tests/cephadm/test_osd_config.py

```diff
@@ -0,0 +1,55 @@
+import os
+import re
+
+from cli.cephadm.cephadm import CephAdm
+from cli.cephadm.exceptions import CephadmOpsExecutionError
+from utility.log import Log
+
+log = Log(__name__)
+
+CEPH_LIB_DIR = "/var/lib/ceph"
+
+
+class MonValueError(Exception):
```

Comment:
Instead of this error, add new exception `MonDaemonError` in https://github.com/red-hat-storage/cephci/blob/master/cli/exceptions.py